pytest.ini

```diff
@@ -0,0 +1,23 @@
+[pytest]
+env =
+    FLASK_ENV=unit_test
+    FLASK_DEBUG=1
```

Comment:
@SentientHousePlant @gio-karunakaran . Do we add 'FORCE_HTTPS = False' in unit-test config (config/environments/unit_test.py) rather than set it in this file? I was forced to do it that way in authenticator because we check that the flask app is not in debug mode when testing and setting that ^ forces the app into debug mode which we cannot overwrite afaik.

Reply:
I don't know if the debug mode test is particularly important? Do other tests fail if it is in debug mode? Could we remove it? - Equally perhaps is it required for any tests to have debug mode on? i.e. do we care either way?

Reply:
@Adam-W1 I had also set 'FORCE_HTTPS = False' in unit-test config and have now removed FLAS_DEBUG=1 in this file and tests are still running fine so we can stick with this? 👍

Reply:
@harryyo ^